HelloWinApp

```diff
@@ -0,0 +1,80 @@
+#include <Windows.h>
+#include <assert.h>
+
+#pragma comment(lib, "kernel32.lib")
+#pragma comment(lib, "user32.lib")
+#pragma comment(lib, "gdi32.lib")
+
+LRESULT CALLBACK WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam);
+
+int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine, int nShowCmd)
+{
+    static TCHAR szClassName[] = TEXT("SHIV-SAN APP");
+    static TCHAR szWindowName[] = TEXT("MY WINDOW"); 
+
+    HWND hWnd = NULL;
+    HBRUSH hBrush = NULL;
+    HICON hIcon = NULL;
+    HICON hIconSm = NULL; 
+    HCURSOR hCursor = NULL;
+
+    WNDCLASSEX wndEx;
+    MSG msg;
+
+    ZeroMemory(&wndEx, sizeof(WNDCLASSEX));
+    ZeroMemory(&msg, sizeof(MSG));
+
+    hBrush = (HBRUSH)GetStockObject(WHITE_BRUSH);
+    assert(hBrush);
+
+    hIcon = LoadIconA(NULL, IDI_APPLICATION);
```

Comment:
```suggestion
    hIcon = LoadIcon(NULL, IDI_APPLICATION);
```

Reply:
Could you change this so that it is same as Line 33

Reply:
correction done.